content/cloud/noops-sql-db.md

```diff
@@ -1,13 +1,11 @@
-title = "NoOps SQL Database"
+title = "NoOps SQL Database Tutorial"
 template = "cloud_main"
-date = "2023-07-26T16:00:00Z"
+date = "2023-10-24T09:00:00Z"
```

Comment:
```suggestion
date = "2023-07-26T16:00:00Z"
```
Putting this date back to original because the future date will make this page `404`. Will add a `only_merge_at ...` label to the PR instead.